.github/workflows/build.yml

```diff
@@ -4,6 +4,11 @@ on:
   pull_request:
     branches:
       - "*"
+  push:
+    branches:
+      - main
```

Comment:
Do we need this? Probably should only run on tag push, not push to main.

Reply:
I was thinking to use the `main` push for verifying merged commits. 
Otherwise, happy to remove it